pages/Documents/ConversationalAI/Third-PartyBots/MicrosoftDirectLine/introduction.md

```diff
@@ -0,0 +1,116 @@
+---
+pagename: Introduction
+redirect_from:
+  - bot-connectors-microsoft-bot-framework.html
+  - third-party-bots-microsoft-bot-framework.html
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: Microsoft Direct Line
+permalink: third-party-bots-microsoft-direct-line-introduction.html
+indicator:
+---
+
+### Overview
+The following documentation outlines the details of the Connector for Microsoft Direct Line.
+It describes how the connector is implemented and what limitations this imposes on bots that are added via this connector.
+It shows how activities need to be composed to successfully send messages and actions to the **Conversational Cloud**.
+
+It does not go into details on implementation details of such a bot.
+Microsoft provides different ways to implement chat bots like the 
+[Bot Framework SDK](https://github.com/microsoft/botframework-sdk#bot-framework-sdk-v4).
+A good starting point is the documentation on 
+[Azure Bot Services](https://docs.microsoft.com/en-us/azure/bot-service/).
+
+### Limitations
+
+{: .important}
+Third Party Bots only supports the Direct Line API version 3.0.
+
+{: .important}
+Third Party Bots only supports authorization with the Direct Line secret. Authorization with a Bot ID and Tenant ID is 
+not supported.
+
+{: .important}
+Third Party Bots uses the replyToId of a bot activity to match it against the consumer message.
```

Comment:
Third-Party Bots

Reply:
Thought Third Party Bots looked better but I changed it throughout the whole documentation now to align to this